src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: inline-grid;
```

Comment:
Consider using flex instead of grid for better compatibility and simpler structure.